mne/io/tests/test_meas_info.py

```diff
@@ -147,3 +148,19 @@ def test_make_dig_points():
                   dig_points[:, :2])
     assert_raises(ValueError, _make_dig_points, None, None, None, None,
                   dig_points[:, :2])
+
+
+def test_create_info():
+    assert_raises(TypeError, create_info, 'Test Ch', 1000, None)
+    assert_raises(ValueError, create_info, ['Test Ch'], -1000, None)
+    assert_raises(ValueError, create_info, ['Test Ch'],  1000, ['eeg', 'eeg'])
+    assert_raises(TypeError, create_info, [np.array([1])], 1000, None)
+    assert_raises(TypeError, create_info, ['Test Ch'], 1000, np.array([1]))
+    assert_raises(KeyError, create_info, ['Test Ch'], 1000, 'awesome')
+    assert_raises(TypeError, create_info, ['Test Ch'], 1000, None,
+                  np.array([1]))
+    m = read_montage('biosemi-32')
+    info = create_info(ch_names=m.ch_names, sfreq=1000., ch_types='eeg',
+                       montage=m)
+    ch_pos = [ch['loc'][:3] for ch in info['chs']]
+    assert_array_equal(ch_pos, m.pos)
```

Comment:
I'm adding a `test_make_info` in my PR that I should have called `test_create_info`. Can you rebase and combine the two sets of tests after mine is merged?